ingestion-beam/src/main/java/com/mozilla/telemetry/options/SinkOptions.java

```diff
@@ -66,6 +74,12 @@
 
   void setErrorOutputType(ErrorOutputType value);
 
+  @Description("Compression format for --errorOutputType=file")
+  @Default.Enum("GZIP")
+  Compression getErrorOutputFileCompression();
+
+  void setErrorOutputFileCompression(Compression value);
+
```

Comment:
The file compression formats can't be ValueProviders due to the FileIO API lacking such an overload.